dependencies/setupAsio.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# $Id$
+
+# Shell script building HopsaGUI dependency Asio automatically
+
+basedir=`pwd`
+name=asio
+codedir=${basedir}/${name}-code
+builddir=${basedir}/${name}-build
+installdir=${basedir}/${name}
+
+# Download and verify
+./download-dependencies.py ${name}
+
+# Copy code to build dir, not sure if out-of-source build is possible
+mkdir -p $builddir
+pushd $builddir
+cp -a $codedir/* .
+
+# Generate makefiles
+chmod u+x ./configure
+./configure --prefix=$installdir --without-boost
```

Comment:
This (configure) will be problematic on Windows, I hope asio and the others have cmake support
I have just recently been able to get rid of the msys2 build dependency, I do not want it back.